modules/local/dnavalidation/main.nf

```diff
@@ -0,0 +1,54 @@
+process DNAVALIDATION {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    'oras://community.wave.seqera.io/library/bash:5.2.21--67f53ad0451dfdce' :
+    'community.wave.seqera.io/library/bash:5.2.21--5bc877f5b6cf0654' }"
+
+    input:
+    tuple val(meta) , path(vcf)
+    tuple val(meta2), path(reads)
```

Comment:
for this part, I would define an input with this structure
```
tuple val(meta), path(vcf), path(reads)
```
because this would give us the room to group all reads (from the case and control samples) together in the same tuple as the VCF and avoid any channel ordering error. otherwise we can't easily control which sample goes in the first input and in the second one